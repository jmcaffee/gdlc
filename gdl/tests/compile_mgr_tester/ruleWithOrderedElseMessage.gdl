/* **************************************************************************
 *  File:
 *
 *  Description:
 *    This is to test a missing rule definition
 *
 ************************************************************************* */


dpm numeric var1;
dpm numeric var2;

rule TestRule()             // Define the rule.
  if(var1 > 100000)
  then
  else
    [4]message(exception, "Test Message");
  end
end

ruleset TestRuleset(true)
  rule TestRule();         // Rule was defined above.
end

