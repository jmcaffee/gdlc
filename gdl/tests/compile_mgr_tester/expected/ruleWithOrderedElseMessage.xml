<Rule Id="1" Name="TestRule"><IfMessages/><ElseMessages><Message Type="Exceptions" Order="4"><![CDATA[Test Message]]></Message></ElseMessages><Condition><Expression><Brace><Compute><Operator>&gt;</Operator><LeftOperand><DPM Name="var1"/></LeftOperand><RightOperand><Constant>100000</Constant></RightOperand></Compute></Brace></Expression></Condition></Rule>